apps/cientos-docs/content/1.getting-started/1.index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Getting Started
+description: Learn how to get started with this documentation boilerplate
+---
+
+Cientos (Spanish word for "hundreds", pronounced /θjentos/) is a collection of useful ready-to-go helpers and components that are not part of the core package. The name uses the word in Spanish to multiply by 100, to refer to the potential reach of the package to hold amazing abstractions.
+
+> **Info**
+>
+> The cientos package uses three-stdlib module under the hood instead of the three/examples/jsm module. This means that you don't need to extend the catalogue of components using the extend method, cientos does it for you.
+
+It just works. 💯
+
+## Installation
+
+```bash [pnpm]
```

Comment:
Blocking:

You can use a code group component here, similar to the other docs https://github.com/Tresjs/tres/blob/dce1b9c880a74ece99751317fc794f98fdfff54e/apps/docs/content/1.getting-started/2.installation.md?plain=1#L14-L26